atom/common/native_mate_converters/blink_converter.cc

```diff
@@ -277,7 +280,7 @@ bool Converter<blink::WebMouseWheelEvent>::FromV8(
   bool can_scroll = true;
   if (dict.Get("canScroll", &can_scroll) && !can_scroll) {
     out->hasPreciseScrollingDeltas = false;
-    out->modifiers &= ~blink::WebInputEvent::ControlKey;
+    out->setModifiers(out->modifiers() | blink::WebInputEvent::ControlKey);
```

Comment:
Is this logic intentionally changed? It is not equivalent. 
Before it keeps all flags except for ControlKey, and now it adds ControlKey flag